### DISCUSSION TITLE
React - Cognito - Google (Invalid params)

### DISCUSSION THREAD

**Discussion by culda** [Web3Auth/Web3Auth#907] [Help]
I'm trying to set up with Cognito and followed everything from here https://web3auth.io/docs/guides/cognito

When I execute login by running `connectTo`, I'm just getting a white screen with some console errors like `OpenLogin - RPC Error: Invalid params`. I have no idea how to start troubleshooting this

web3auth Version: 3.

![2022-11-19-175322](https://user-images.githubusercontent.com/48595067/202865096-08d5d6d9-e190-4802-9c1c-8e5a25a8e6d6.png)
0.4

**Comment by shahbaz17** [comment]
Hey @culda 

Please share the code snippet of web3auth initialisation and connectTo function. Also the verifier details.

**Reply by culda** [reply]
@shahbaz17 


```
    const provider = await web3Auth.connectTo(WALLET_ADAPTERS.OPENLOGIN, {
      loginProvider: 'jwt',
      extraLoginOptions: {
        domain: 'https://rugzytest.auth.us-east-1.amazoncognito.com',
        verifierIdField: 'email',
        response_type: 'token',
        scope: 'email profile openid',
      },
    })
```

```
    const web3authCore = new Web3AuthCore({
      clientId: process.env.REACT_APP_WEB3AUTH_CLIENT_ID as string,
      chainConfig: {
        chainNamespace: CHAIN_NAMESPACES.EIP155,
        chainId: '0x1',
        rpcTarget: 'https://eth-mainnet.g.alchemy.com/v2/ZL5pl2MBIJ7djN6oecDb5rs4z--2bzYl',
        blockExplorer: 'https://etherscan.io/',
        ticker: 'ETH',
        tickerName: 'Ethereum',
      },
    })
    const loginAdapter = new OpenloginAdapter({
      adapterSettings: {
        network: 'testnet',
        clientId: process.env.REACT_APP_WEB3AUTH_CLIENT_ID as string,
        uxMode: 'popup',
        loginConfig: {
          jwt: {
            name: 'any name',
            verifier: 'cognito-verifier',
            typeOfLogin: 'jwt',
            clientId: '360978832945-vj1g7hcm8mn3vfn6mi6o1g4dmplmn5jn.apps.googleusercontent.com',
          },
        },
      },
    })
    web3authCore.configureAdapter(loginAdapter)
```

**Reply by culda** [reply]
some images from cognito
![image](https://user-images.githubusercontent.com/48595067/202894704-fc576209-f10c-4333-affc-02f33aad310a.png)

![image](https://user-images.githubusercontent.com/48595067/202894669-f258a51e-8a4f-4319-8457-7428838f5694.png)

**Reply by culda** [reply]
I checked the cognito example on github and fixed a couple of config mistakes. I've got some JS errors now which I'm working on, unrelated to web3auth. I'll report back once I've worked them out

**Reply by culda** [reply]
Correction, it is related, I'm getting exactly this error https://github.com/Web3Auth/web3auth-web/discussions/853
I'm on react-scripts 4.0.3

**Reply by shahbaz17** [reply]
So, can you try the solution provider https://github.com/orgs/Web3Auth/discussions/853#discussioncomment-4158402? 

And please check https://web3auth.io/docs/troubleshooting/webpack-issues too

**Reply by culda** [reply]
I couldn't work out webpack 4, so I upgraded to webpack 5 and the fix above worked!
